File: 250402/A 부터 B 까지/a-to-b.py
A, B = map(int, input().split())
result = [A]

while A < B:
    if A % 2 == 1:
        A = A * 2
    else:
        A = A + 3
    
    if A > B:
        break
    result.append(A)

print(*result, end=' ')

File: 250922/A 부터 B 까지/a-to-b.js
const fs = require("fs");
let [A, B] = fs.readFileSync(0).toString().split(" ").map(Number);

let result = "";
let i = A;

while (i <= B) {
    result += (i + " ");
    if (i % 2 === 1) {
        i *= 2
    } else {
        i += 3
    }
};

console.log(result);